Index: src/drivers/acpi/nfit/core.c
===================================================================
--- src.orig/drivers/acpi/nfit/core.c	2019-01-31 14:14:14.153075974 +0100
+++ src/drivers/acpi/nfit/core.c	2019-01-31 14:43:59.408176302 +0100
@@ -57,6 +57,8 @@
 LIST_HEAD(acpi_descs);
 DEFINE_MUTEX(acpi_desc_lock);
 
+#define NFIT_VERSION "3.10.0-957_dup7.6"
+
 static struct workqueue_struct *nfit_wq;
 
 struct nfit_table_prev {
@@ -3570,3 +3572,4 @@
 module_exit(nfit_exit);
 MODULE_LICENSE("GPL v2");
 MODULE_AUTHOR("Intel Corporation");
+MODULE_VERSION(NFIT_VERSION);
